.github/workflows/ci.yml

```diff
@@ -28,6 +33,10 @@ jobs:
           java-version: '17'
           cache: sbt
 
+
+      - name: Install sbt
+        run: sudo curl -Lo /usr/local/bin/sbt https://raw.githubusercontent.com/sbt/sbt/refs/heads/1.10.x/sbt && sudo chmod +x /usr/local/bin/sbt
```

Comment:
Let's use https://github.com/sbt/setup-sbt?tab=readme-ov-file instead

Reply:
Oh, I have even better idea. Why don't we just add a sbt script to the repo instead. They can never take that away from us! :muscle: 
What do you think?

Reply:
I'd rather not – I personally dislike launcher scripts, I think they are an anti-pattern, caused by deficiencies in the availability of fundamental tools (such as SBT). Only case where I think a launcher script is necessary is when you have some strong requirements for the JVM/flags for sbt build, e.g. https://github.com/sourcegraph/scip-java/blob/main/sbt

The Github Actions are the ones behaving "incorrectly" here, so I'd rather have the workaround (setup-sbt action) present only in the workflow scripts. For local development I assume people will have the SBT installed.

Reply:
> I personally dislike launcher scripts...

I love them :D They make the project buildable/CIable reliably and universally. But fair enough.

Reply:
> reliably and universally

As long as you have two – one for unix and one for windows :P 

Cargo, NPM, Bazel, Maven, many many other builds don't have launcher scripts.

The strange fascination with launcher scripts among some JVM tools (Gradle, Mill, SBT) is the exception, not the rule – and the various discrepancies over the years in the launcher scripts have shown that checking in a bunch of _someone else's_ brittle logic is not bullet proof.

Anyways, I'm aware of my blasphemous position on this, but I can't help but being right :D